ledger2beancount.yml

```diff
@@ -39,6 +52,17 @@ commodity_map:
 altdate_tag: aux-date
 code_tag: code
 
+# Render tags as tags:
+# If true, tags are rendered as #tags after the narration
+# If false, tags are stored as meta data with the key "tags"
+tag-as-tag: true
```

Comment:
This is name nitpicking, but I think "tag as tag" should be the default behavior, as such it would be more natural to have an option that *changes* that natural behavior. How about `tag_as_metadata` instead, defaulting to `false`?